packages/react-dom/src/__tests__/ReactUpdates-test.js

```diff
@@ -1064,28 +1223,18 @@ describe('ReactUpdates', () => {
       }
     }
 
-    ReactDOM.render(<App />, document.createElement('div'));
-  });
-
-  it('unstable_batchedUpdates should return value from a callback', () => {
-    const result = ReactDOM.unstable_batchedUpdates(function () {
-      return 42;
+    const root = ReactDOMClient.createRoot(document.createElement('div'));
+    await act(() => {
+      root.render(<App />);
     });
-    expect(result).toEqual(42);
-  });
 
-  it('unmounts and remounts a root in the same batch', () => {
```

Comment:
Deleted, because `createRoot` does not support rendering to an unmounted root. The `.render` version exists in `ReactLegacyUpdates` now, and we have a test for the error rendering to an unmounted container elsewhere.